src/components/Books.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Books = () => (
+  <div>
+    <h4>List of books</h4>
+    <li>
+      <button type="button">Remove</button>
+    </li>
+    <form>
+      <input type="text" name="title" placeholder="Book Title" />
+      <input type="text" name="author" placeholder="Book Author" />
+      <button type="submit">Add Book</button>
+    </form>
```

Comment:
- [ ] This form should be in a separate component, kindly implement this change 😄
   - Maybe a `BookForm` component 🚀